docs/dev/metrics.md

```diff
@@ -115,16 +115,23 @@ mapi_instance_delete_failed{name=machine-3,namespace=openshift-machine-api,reaso
 ## Metrics about MachineHealthCheck resources
 
 When using MachineHealthChecks, metrics are available from the `machine-api-controllers` Pod on the
-default metrics port(`8083`) for the `machine-healthcheck-controller` container. 
+default metrics port(`8083`) for the `machine-healthcheck-controller` container.
 
 The `mapi_machinehealthcheck_nodes_covered` metric describes the number of Nodes that are currently
 being monitored by `machine-healthcheck-controller`. The `name` label in this metric refers to the
 name of the MachineHealthCheck that is being reported.
 
+The `mapi_machinehealthcheck_remediation_success_total` metric gives a total count of the successful
+remediation performed by a MachineHealthCheck. The `name` label in this metric refers to the name
+of the MachineHealthCheck that is being reported.
```

Comment:
Do we need a namespace label? Is it possible to have MHCs in multiple namespaces in a cluster? Does it hurt having this extra dimension?